src/components/Dashboard/FilingHistoryList/FutureEffective.vue

```diff
@@ -44,23 +46,20 @@ export default class FutureEffective extends Mixins(DateMixin) {
       return {
         subtitle: 'Future Effective Incorporation Date',
         filingLabel: 'incorporation',
-        companyLabel: (this.getEntityName || 'this Numbered Benefit Company'),
```

Comment:
I changed this because we will soon support CP IA, in other words, `isFutureEffectiveIa` could be true, but the company label would be wrong. The company label is now coded (in the HTML) as the entity name (if if exists) or "this company".

Reply:
Oops, I need to change this in FutureEffectivePending.vue as well... next commit.